bin/pull-from-gcr.sh

```diff
@@ -21,7 +21,8 @@ IMAGES=(
     crunchy-prometheus
     crunchy-scheduler
     crunchy-upgrade
-    crunchy-pgbasebackup-restore
+    crunchy-pgbasebackup-restore.
+    crunchy-postgres-ha
```

Comment:
@andrewlecuyer `crunchy-postgres-gis-ha` too?

Reply:
`crunchy-postgres-gis-ha` is now included